### ISSUE TITLE
[AIR][Tune] Provide user guide on how to build active learning on AIR

### ISSUE THREAD

**Issue by jianoaix** [ray-project/ray#30157]
We have users building active learning in the following pattern:
- Create 3 datasets: train_ds, eval_ds, active_pool_ds
- Train a base model with <train_ds, eval_ds>
  - Load a pretained model (e.g. HuggingFace transformer)
  - Train it with <train_ds, eval_ds>
  - Save the trained model
- Run N trials:
  -  Create a trial_train_ds by sampling active_pool_ds
  - Train a model from base model, with <trial_train_ds, eval_ds>
    - Load the base model trained above
    - Train it with <trial_train_ds, eval_ds>
    - Save the trained model

This use case involves the active learning + pre-tained model fine-tuning. In particular, it will use a feature in Tune which treats Dataset as hyperparameters.

cc @xwjiang2010 @Yard1

**Comment by Hangzhi** [comment]
Hi @jianoaix, can I take this issue? May I have a pointer to where the doc should be placed?

**Comment by jianoaix** [comment]
Yep, thank you @Hangzhi for taking this up! You may place it under docs.ray.io/en/master/ray-air/examples.

**Comment by cszhu** [comment]
This P2 issue has seen no activity in the past 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please comment and remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
